tests/behat/features/capabilities/event/all-day-timezone.feature

```diff
@@ -44,3 +44,36 @@ Feature: All day event for different timezones
     And I should see "THIS IS A TEST EVENT - EDIT"
     And I should not see "11:00"
     And I should not see "31 Dec '24"
+
+  @verified @perfect @critical
```

Comment:
Thanks for taking the time to write a test @vcsvinicius, well done on your first effort! 🎉 
I think what we can do here is simplify it quite a lot; what you're actually testing here is the creation of an event, by going to `node/add/event` all those things are already tested in separate steps, doing it like this could mean your test fails when the event creation form is broken and that is not the goal of your test.

For that you can use something similar to:
https://github.com/goalgorilla/open_social/blob/main/tests/behat/features/capabilities/event/eventenrollment-manage.feature#L11

Where you start with already created events prepared for your scenario to run and the only thing you need is to prepare them with the correct start / end date 

and then to finish it of with your last steps

```
When I am on "community-events"
    Then I should see "This is a timezone test for all day events"
    And I should see "1 Jan '35 - 2 Jan '35"
```

Because that is what you ultimately only want to test